MANIFEST.in

```diff
@@ -9,6 +9,7 @@ exclude .pylintrc
 exclude tox.ini
 exclude requirements.txt
 exclude requirements-dev.txt
+exclude requirements/*
```

Comment:
```suggestion
exclude requirements/
```

Reply:
trailing backslashes in this doesn't work on windows and causes a warning. windows doesn't matter, but the warning does.

Reply:
oh really? then sorry for the noise :)